### ISSUE TITLE
Git log with custom format - incorrect escaping

### ISSUE THREAD

**Issue by asvetliakov** [andy-5/wslgit#46]
This command
```
wslgit.exe log --name-status --format="%x3c%x2ff%x3e%n%x3cr%x3e %H%n%x3ca%x3e %an%n%x3ce%x3e %ae%n%x3cd%x3e %at%n%x3cp%x3e %P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e" -n200 --follow -- packages/backendjs/src/processing/portal/chatHandlers/IgnoreChat.ts
```

Results to:
```
fatal: bad revision '"--format=%x3c%x2ff%x3e%n%x3cr%x3e %H%n%x3ca%x3e %an%n%x3ce%x3e %ae%n%x3cd%x3e %at%n%x3cp%x3e %P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e"'
```

VSCode GitLens plugin is using such custom formats for log & stash commands

**Comment by asvetliakov** [comment]
Yes, without spaces it works normally (after merging rev-parse fix)

**Comment by diablodale** [comment]
@asvetliakov , in your specific computer/wslgit setup, are you using the advanced feature where you have to set either `WSLGIT_USE_INTERACTIVE_SHELL` and/or `BASH_ENV` ?

I ask because I have isolated a behavior related to it and the escaping of spaces.

**Comment by asvetliakov** [comment]
@diablodale Yes, i'm using ```WSLGIT_USE_INTERACTIVE_SHELL=false```. However this doesn't seem to change anything

**Comment by diablodale** [comment]
I found some issues and I've made a fix and pull request.
@asvetliakov , compile the code in my PR that should link here. Or if you don't have that ability and trust, here is a release compile of the same code.
[wslgit.zip](https://github.com/andy-5/wslgit/files/2475987/wslgit.zip)

I tested this with your original test case above - it works now.
I also tested this against six combinations including spaces in working directory and parameters.
I would appreciate feedback if this works better for you both.
